docs/api/duckdb/multi-database-support.md

```diff
@@ -0,0 +1,157 @@
+  vg.loadParquet("ca55", "http://localhost:5173/data/ca55-south.parquet"),   // Load the dataset into the PostgreSQL database
+]);
+
+// create interactive visualizations for your dataset
+const $interp = vg.Param.value("random-walk");
+const $blur = vg.Param.value(0);
+
+export default vg.vconcat(
+  vg.hconcat(
+    vg.menu({
+      label: "Interpolation Method",
+      options: ["none", "nearest", "barycentric", "random-walk"],
+      as: $interp
+    }),
+    vg.hspace("1em"),
+    vg.slider({label: "Blur", min: 0, max: 100, as: $blur})
+  ),
+  vg.vspace("1em"),
+  vg.plot(
+    vg.raster(
+      vg.from("ca55"),
+      {
+        x: "LONGITUDE",
+        y: "LATITUDE",
+        fill: vg.max("MAG_IGRF90"),
+        interpolate: $interp,
+        bandwidth: $blur
+      }
+    ),
+    vg.colorScale("diverging"),
+    vg.colorDomain(vg.Fixed)
+  )
+);
+
+```
+
+
+### Connecting to `MySQL`:
+
+``` js
+import { vg, clear } from "http://localhost:5173/dev/setup.js";
+
+clear();
+
+const password = "YOUR_PASSWORD";
+
+const databaseName = "YOUR_DATABASE_NAME";
+
+const mysql_connection_string = `host=localhost password=${password} user=root port=0 database=${databaseName}`;
+
+await vg.coordinator().exec([
+  "INSTALL mysql;",   // Install MySQL extension
+  "LOAD mysql;",   // Manually load MySQL extension
+  `ATTACH '${mysql_connection_string}' AS mysqldb (TYPE MYSQL);`,   // Attach the MySQL database
+  "USE mysqldb;",   // Specify the database name
+  vg.loadParquet("ca55", "http://localhost:5173/data/ca55-south.parquet"),   // Load the dataset into the MySQL database
+]);
+
+// Create interactive visualizations for your dataset
+const $interp = vg.Param.value("random-walk");
+const $blur = vg.Param.value(0);
+
+export default vg.vconcat(
+  vg.hconcat(
+    vg.menu({
+      label: "Interpolation Method",
+      options: ["none", "nearest", "barycentric", "random-walk"],
+      as: $interp
+    }),
+    vg.hspace("1em"),
+    vg.slider({label: "Blur", min: 0, max: 100, as: $blur})
+  ),
+  vg.vspace("1em"),
+  vg.plot(
+    vg.raster(
+      vg.from("ca55"),
+      {
+        x: "LONGITUDE",
+        y: "LATITUDE",
+        fill: vg.max("MAG_IGRF90"),
+        interpolate: $interp,
+        bandwidth: $blur
+      }
+    ),
+    vg.colorScale("diverging"),
+    vg.colorDomain(vg.Fixed)
+  )
+);
+```
+
+### Connecting to `SQLite`:
+
+``` js
+import { vg, clear } from "http://localhost:5173/dev/setup.js";
+
+clear();
+
+await vg.coordinator().exec([
+  "INSTALL sqlite;",   // Install SQLite extension
+  "LOAD sqlite;",   // Load SQLite extension
+  `ATTACH 'sakila.db' (TYPE SQLITE);`,   // Attach SQLite database
+  "USE sakila;",   // Switch to the database called sakila
+  vg.loadParquet("ca55", "http://localhost:5173/data/ca55-south.parquet"), // Load the dataset into the SQLite database
+]);
+
+// Create interactive visualizations for your dataset
+const $interp = vg.Param.value("random-walk");
```

Comment:
I don't think we need to repeat the specification for all three databases. I think including it once (the first time) should be plenty. Then the rest of the example could just be a comment like `// Mosaic specification goes here` or similar.